src/main/java/data/JsonConverter.java

```diff
@@ -74,26 +74,33 @@ public File writeFile(String vendorID, String jsonConfig) {
 
   /**
    * Retrieve and return the contents of the desired configuration.
+   * @throws FileNotFoundException thrown when configuration is not found.
    */
-  public String getConfig(String vendorID) {
+  public String getConfigText(String vendorID) throws FileNotFoundException {
     String configContents = "";
-
     // Retrieve the file with the corresponding vendorID.
     File config = new File(basePath + vendorID);
 
-    Scanner input;
-    try {
-      input = new Scanner(config);
-    } catch(FileNotFoundException e) {
-      return null;
-    }
+    Scanner input = new Scanner(config);
 
     while (input.hasNextLine()) {
       configContents += input.nextLine();
     }
     return configContents;
   }
 
+  public String getAccountConfig(String vendorId, String accountId)
+          throws IllegalArgumentException, IOException {
+
+    String config = getConfigText(vendorId);
+    Vendor vendor = new Vendor(config, vendorId);
+    Account account = vendor.getAccountById(accountId);
+    if (account == null) {
+      throw new IllegalArgumentException();
```

Comment:
nit: Include an error message in this exception. Something like "account with ID %s not found"